packages/airnode-deployer/src/infrastructure/index.test.ts

```diff
@@ -441,22 +442,20 @@ describe('deployAirnode', () => {
   });
 
   it('deploys Airnode', async () => {
-    const terraformOutput = await infrastructure.deployAirnode(config, configPath, secretsPath);
+    const terraformOutput = await infrastructure.deployAirnode(config, configPath, secretsPath, Date.now());
```

Comment:
Shouldn't the date be mocked somehow?

Reply:
I don't think it needs to be for these tests, do you?

Reply:
No, I was just wondering whether it doesn't cause some issues, but the date is probably irrelevant for that test.